### PR TITLE
Unify interface with charter-manager

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/CropManager.sol [hunk 1]
```diff
@@ -171,13 +171,11 @@ contract CropManagerImp {
         VatLike(vat).move(urp, dst, rad);
     }
 
-    function frob(address crop, address u, address v, address w, int256 dink, int256 dart) external allowed(u) allowed(w) {
-        require(VatLike(vat).wards(crop) == 1, "CropManager/crop-not-authorized");
-
+    function frob(bytes32 ilk, address u, address v, address w, int256 dink, int256 dart) external allowed(u) allowed(w) {
         require(u == v, "CropManager/not-matching");
```
Comment (hexonaut) [inline src/CropManager.sol:175]: Could we add a note on this that this require can never be relaxed? Otherwise the `tack` could lose track of the rewards.

Comment (talbaneth) [reply src/CropManager.sol:176]: Done.

**Review by hexonaut — APPROVED** [review]
LGTM

**Review by gbalabasquer — APPROVED** [review]
LGTM

**Pull request merged**